docs/sidebars.js

```diff
@@ -314,7 +314,7 @@ const sidebars = {
         id: "dev_docs/wallets/main",
       },
       items: [
-        "dev_docs/wallets/building_a_wallet",
+        "dev_docs/wallets/architecture",
```

Comment:
same as this comment https://github.com/AztecProtocol/aztec-packages/pull/1859/files#r1309357399